integrations/tensorflow/bindings/python/pyiree/tf/support/tf_utils.py

```diff
@@ -302,7 +297,18 @@ def supports_cxx_serialization() -> bool:
     return True
 
 
-class _TfFunctionWrapper(object):
+def _normalize_numpy(result: np.ndarray):
+  """Normalizes TF/Lite's output to match IREE's"""
```

Comment:
TFLite, not TF/Lite.

Reply:
Whoops, I intended for that to mean TF and TFLite.